localstack-core/localstack/services/apigateway/next_gen/execute_api/router.py

```diff
@@ -72,14 +72,14 @@ def prepare_rest_api_invocation(
         return context, response
 
     def is_rest_api(self, api_id: str, stage: str):
-        return (api_id, stage) in self._global_store.active_deployments
+        return stage in self._global_store.active_deployments.get(api_id)
```

Comment:
sorry again, this will raise `TypeError: argument of type 'NoneType' is not iterable`, sorry for the back and forth on this 😅

Reply:
:man_facepalming: This really is our day :rofl: